deployment/ansible/roles/model-my-watershed.geotrellis/tasks/compile.yml

```diff
@@ -0,0 +1,6 @@
+---
+- name: Compile the scala code
+  shell: "sbt 'project services' compile assembly"
```

Comment:
No, just forgot about command. Fixed in 89a5306013238db270075965a62c3a5f0253fdb8